examples/bobandlynn.jsonld

```diff
@@ -0,0 +1,144 @@
+{
+    "@context": {
```

Comment:
Also, I am using rdfs:label.  I guess we need to finish the discussion on standardizing the labels, and I can change this then.